Changes

```diff
@@ -710,6 +710,10 @@ Working version
 - #12831: Fix call to caml_call_realloc_stack for s390x in PIC mode
   (Vincent Laviron, report by Jerry James, review by Miod Vallat)
 
+* #12837: Show non-generalizable type parameters in type definitions
```

Comment:
This change is only breaking the compiler API, I think it would be clearer for most people to mark it as non-breaking.

Reply:
Indeed, I don't know of people in the wild who depend on the outcometree type definition -- in any sort of supported way.